variables.tf

```
@@ -86,6 +86,12 @@ variable "github_token" {
   description = "(Optional) GitHub auth token environment variable (`GITHUB_TOKEN`)"
 }
 
+variable "github_token_type" {
+  type        = string
+  default     = "SECRETS_MANAGER"
```

At Cloud Posse, we've used ssm param store as our default. I think we should be consistent here and keep it the same. If someone wants to override it then they can use this input vadiable to set it to use ssm secrets manager.

```suggestion
  default     = "PARAMETER_STORE"
```